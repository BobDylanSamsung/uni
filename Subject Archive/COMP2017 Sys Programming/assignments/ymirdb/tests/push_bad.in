set a 1 2 3
push a b
get a
push a b c d
get a
list keys
push b a
list entries
set b 1
push 1a b
bye
