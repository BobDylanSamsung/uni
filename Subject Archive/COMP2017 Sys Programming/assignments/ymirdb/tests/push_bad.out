ok

no such key

[1 2 3]

no such key

[1 2 3]

a

no such key

a [1 2 3]

ok

no such key

bye
